components/tools/ToolDashboard.js

```diff
@@ -123,7 +139,22 @@ export default function ToolDashboard() {
                         )}
                     </div>
                 </div>
-                <div className="py-1 px-4 flex rounded-lg border w-2/3 lg:w-4/5 border-gray-300 hover:border-gray-600 focus:border-gray-600 text-gray-700 shadow text-sm">
+                <div className='flex w-[48%] lg:w-1/5 gap-5 h-auto'>
+                    <div className="relative w-full h-auto" ref={categoryRef}>
+                        <div
+                            className="flex py-1 px-4 justify-center items-center gap-2 rounded-lg border w-full h-14 border-gray-300 hover:shadow-md hover:border-gray-600 text-gray-700 shadow text-sm cursor-pointer"
+                            onClick={() => setopenCategory(!openCategory)}>
+                            <div>Jump to Category</div>
+                            <ArrowDown className={`my-auto ${openCategory ? 'rotate-180' : ''}`} />
+                        </div>
+                        {openCategory && (
+                            <div className="z-20 absolute top-16">
```

Comment:
```suggestion
                            <div className="z-20 absolute top-16 right-52 lg;right-0">
```

I think this will correct the scroll error.

Reply:
`lg:right-0` is making the dropdown appear below the search bar for large screens. I think `right-52` is good for all screen sizes.